lib/constants.js

```diff
@@ -98,11 +98,11 @@ module.exports = {
   REGEX_REMOVE_BACKSLASH: /(?:\[.*?[^\\]\]|\\(?=.))/g,
 
   // Replace globs with equivalent patterns to reduce parsing time.
-  REPLACEMENTS: {
+  REPLACEMENTS: Object.create(null, {
     '***': '*',
     '**/**': '**',
     '**/**/**': '**'
-  },
+  }),
```

Comment:
The second argument of `Object.create` is an object of _property descriptors_, not raw values. So if we want to go this route, it should look like this:
```js
REPLACEMENTS: Object.create(null, {
  '***': { value: '*', enumerable: true },
  '**/**': { value: '**', enumerable: true },
  '**/**/**': { value: '**', enumerable: true }
})
```

I still think the literal syntax is cleaner, but that's just a personal preference:
```js
REPLACEMENTS: {
  __proto__: null,
  '***': '*',
  '**/**': '**',
  '**/**/**': '**'
}
```